assets/payment/settings_script.js

```diff
@@ -16,6 +17,10 @@ function confirmPlanChange(){
 	BuyLicenseWindow.close();
 	confirmPlanChangeWindow();
 }
+function customStripeForm(planRank){
```

Comment:
Codacy found an issue: ['customStripeForm' is defined but never used.](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5864152)